packages/react-hooks/src/useBlockTime.ts

```diff
@@ -28,17 +28,17 @@ export function calcBlockTime (blockTime: BN, blocks: BN, t: TFunction): Result
     `${value < 0 ? '+' : ''}${[
       days
         ? (days > 1)
-          ? t<string>('{{days}} days', { replace: { days } })
-          : t<string>('1 day')
+          ? t<string>('{{days}} d', { replace: { days } })
+          : t<string>('1 d')
```

Comment:
We can actually combine these now since we don't have `day` & `days`, but only `d`. GH won't let me comment, but we could simplify -

```
[
  days
    ? t<string>('{{days}} d', { replace: { days } })
    : null,
  hours
    ? t<string>('{{hours}} h', { replace: { hours } })
    : null
...
]
```

Reply:
I initially simplified it to that, but I reread the condition that it was `days > 1`, so I didn't know if `days` is ever a decimal amount less than 1, where we'd want to display it as **1**, which is why I kept it as is. I can simplify it if that's not the case.

Reply:
It is indeed an integer.